Update todoStorage.js

githubERIK opened the pull request:
I experimented with the code and Deployd backend (http://deployd.com/). It seemed to me that clearCompleted function worked better with my proposed code.

samccone commented:
@githubERIK looks like a few style issues on CI would you mind fixing those up

githubERIK commented:
Managed to fix those style errors.

passy commented:
@githubERIK Could you explain why you want to change the code like this?

Also, it seems like you have found a loophole in our style guide, because this still looks very foreign compared to other code in that file and elsewhere. :)

githubERIK commented:
With the master branch code I got "DELETE (my_localhost_URL_ path) 400 (Bad Request)" 
and the function did not delete records from Delpoyd and did not update the view. I changed the code and it seemed that it started to work. However, I just now noticed it will delete  only one record. Because of that, I now changed the code again and it seems to work well (_It will delete  all completed records_).
All in all, there is a chance that the original code in master branch did not work for me beacuse I might have modified my todoStorage.js http-paths the wrong way.

addyosmani commented:
Travis still seems to be complaining here :monkey:

githubERIK commented:
Indeed. Could somebody explain me, what Travis means by the following output?

```
...
/home/travis/build/tastejs/todomvc/tests/node_modules/drool/lib/index.js:27
  return (new webdriver.WebDriver.Logs(driver))
          ^
TypeError: webdriver.WebDriver.Logs is not a function
...
```

githubERIK commented:
Pull request #1606 fixed Travis `webdriver.WebDriver.Logs` error. The Travis CI build passed.

arthurvr commented on examples/angularjs/js/services/todoStorage.js:
```diff
@@ -34,18 +34,12 @@ angular.module('todomvc')
-					});
+				for (var i = 0; i < completeTodos.length; i++) {
+					store.delete(completeTodos[i]);
+				};
```
No need to have a semicolon here. Kinda surprises me that the linter doesn't check for this? :/

arthurvr commented on examples/angularjs/js/services/todoStorage.js:
```diff
@@ -34,18 +34,12 @@ angular.module('todomvc')
-					}, function error() {
-						angular.copy(originalTodos, store.todos);
-					});
+				for (var i = 0; i < completeTodos.length; i++) {
```
Just use `forEach`? :)
